sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -800,6 +801,10 @@ void CAFMaker::produce(art::Event& evt) noexcept {
     }
   }
 
+  if (fParams.FoldEMShowerDaughters()) {
+    true_particles = caf::FoldEMShowerDaughters(true_particles);
+  }
```

Comment:
Is this a no-op if the daughters are already folded at the geant level? That would actually be a good test one could do relatively simply using `diff_cafs`